src/homepageExperience/components/steps/Overview.tsx

```diff
@@ -1,39 +1,59 @@
-import React, {PureComponent} from 'react'
+import React, {FC, useEffect, useRef} from 'react'
 import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+import {event} from 'src/cloud/utils/reporting'
 
-export class Overview extends PureComponent {
-  render() {
-    return (
-      <div>
-        <h1>Hello, Time-Series World!</h1>
-        <article>
-          <p>Welcome and thanks for checking out InfluxData!</p>
+type Props = {
+  wizard: string
+}
+
+export const Overview: FC<Props> = ({wizard}) => {
+  const videoFrame = useRef<null | HTMLIFrameElement>(null)
+
+  // iFrames represent separate browsing context, requiring this workaround. For more, see resolution of this ticket (https://github.com/influxdata/ui/issues/4623)
 
-          <p>
-            In the next 5 minutes, you will set up InfluxData on your machine
-            and write and execute some basic queries.
-          </p>
+  useEffect(() => {
+    const checkVidClick = () => {
+      if (
+        document.activeElement &&
+        document.activeElement.nodeName.toLowerCase() === 'iframe' &&
+        videoFrame.current &&
+        videoFrame.current === document.activeElement
```

Comment:
Gold star for checking on existence before using it! ⭐

But we have access to [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) so this can be shortened to

```ts
document?.activeElement?.nodeName?.toLowerCase() === 'iframe'
```
and
```ts
videoFrame?.current === document.activeElement
```